packages/core/src/common.css

```diff
@@ -17,3 +17,34 @@
 		padding: 0;
 	}
 }
+
+:host([busy]) {
```

Comment:
The idea is when the the components has an attribute `busy`. I will show the busy status. such as `<ts-button busy="something"></ts-button>`. If you the component doesn't have the `busy` props, it won't show the animation bar.

Reply:
Since all the components are extended from `core` and you added `busy` props to the (core)[https://github.com/Tradeshift/elements/pull/79/files#diff-1f0f1f41e095f13c9f2f24900f28fadfR18] it actually affects all.

Reply:
Move the code to buttons.